src/main/java/com/exasol/dbbuilder/dialects/Table.java

```diff
@@ -146,7 +146,9 @@ public Builder column(final String columnName, final String columnType) {
          * @return new {@link Table} instance
          */
         public Table build() {
-            return new Table(this);
+            final Table table = new Table(this);
+            this.writer.write(table);
```

Comment:
Do we need to check here whether the parent object is marked as "deleted" already?

Reply:
Good point! But it's not necessary because calling `schema.createTable(...)` already checks if the schema was deleted.